# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/hppa-no_setjmp.patch
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

No suprious _setjmp symbol, to fix pdksh.

  - Rene Rebe <rene@exactcode.de>

ld: main.o: in function `unwind':
undefined reference to `_longjmp'

--- dietlibc-0.33/parisc/setjmp.S.vanilla	2018-08-15 20:08:05.642636829 +0000
+++ dietlibc-0.33/parisc/setjmp.S	2018-08-15 20:08:27.826636495 +0000
@@ -18,12 +18,9 @@
 
 .weak setjmp
 .type setjmp,@function
-.globl _setjmp
-.type _setjmp,@function
 .globl __setjmp
 .type __setjmp,@function
 setjmp:
-_setjmp:
 __setjmp:
 	ldi 0, %r25
 
